README.md

```diff
@@ -1,5 +1,8 @@
 # kempctl
 
+[![Go Report Card](https://goreportcard.com/badge/github.com/giantswarm/kempctl)](https://goreportcard.com/report/github.com/giantswarm/kempctl) [![Godoc](https://godoc.org/github.com/giantswarm/kempctl?status.svg)](http://godoc.org/github.com/giantswarm/kempctl)
```

Comment:
Why don't you do at the same time? What is the blocker to not do it in the same issue ?